query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/top_level_mutations/update_many.rs

```diff
@@ -409,7 +409,7 @@ mod json_update_many {
                 }
               }"#,
             2009,
-            "Enum value 'AnyNull' is invalid for enum type NullableJsonNullValueInput"
+            "`AnyNull` is not a valid `NullableJsonNullValueInput`"
```

Comment:
Any reason we've removed the `"Enum value"`characterization?

Reply:
Generalization, we use the same factory for any value not belonging to a type. Generating a message for enums would have required a different factory or a more sophisticated analysis of the types. Given this error message is only internal to the engine and that the client will generate its own error message from the metadata, I thought it was good enough.